src/tests/bulkConcepts/containers/BulkConceptsPage.test.js

```diff
@@ -225,18 +225,14 @@ describe('Test suite for BulkConceptsPage component', () => {
     expect(mapStateToProps(initialState).preview).toEqual([]);
     expect(mapStateToProps(initialState).loading).toEqual(false);
   });
-  it('should show all concepts', () => {
+  it('should render with at least one concept', () => {
```

Comment:
Where does this one concept come from?

Reply:
Mock file containing mock concepts.

Reply:
I thought the tested behaviour is for the screen not to load any concepts until when one types a filter.

Reply:
Sorry about that this was from the previous implementation when I was fetching concepts on componentDidMount. I have removed it.

Reply:
So why does it reflect as part of a commit for this ticket?

Reply:
It is not supposed to be there.